src/Game.coffee

```diff
@@ -243,11 +244,20 @@ class Game
     @sendMessage redraw, @, @
     @score += POINTS_PER_COIN
     @updateScore()
+    # perfectly counts and updates remaining coins on the board
+    #@board.revealPitTiles()
+    console.log("Coins Remaining: #{@board.coinsRemaining()}")
     if @board.coinsRemaining() <= 0
-      @board.revealAll()
-      @sendMessage redraw, @, @
-      # TODO - [rmarks] reset the board
+      @board.revealPitTiles()
```

Comment:
this works, but should probably be refactored into a send message and associated handler